phone/src/ui/components/GridMenu.tsx

```diff
@@ -35,25 +43,23 @@ export const GridMenu = ({ items, Component = AppIcon }) => {
         {items &&
           items.length &&
           items.map((item) => (
-            // <Tooltip
-            //   title={item.id}
-            //   placement='top'
-            //   arrow
-            //   classes={{ tooltip: classes.tooltip }}
-            //   PopperProps={{
-            //     anchorEl={}
-            //   }}
-            // >
-            <Grid
+            <Tooltip
```

Comment:
Portals are broken when using the CSS zoom property, as React's underlying render is not updated to match the dimensions of the DOM element in which the property `zoom` is applied to. See the following image for an example:

![Dom Layout](https://i.tasoagc.dev/eO7B)